### ISSUE TITLE
Add docs about the WiFi-Box in this repo's wiki

### ISSUE THREAD

**Issue by companje** [Doodle3D/WiFi-Box#10]
https://github.com/Doodle3D/WiFi-Box/wiki

**Comment by companje** [comment]
This is a drawing explaining how the WiFi-Box connects drawing apps on different devices over WiFi to 3D-printers through USB.

PNG:
![doodle3d wifi-box setup 2](https://cloud.githubusercontent.com/assets/156066/15449296/fa7fab0e-1f79-11e6-9c98-e4b48b0815d0.png)

**Comment by companje** [comment]
This image explains shows how a drawing made in the Doodle3D app is send to the WiFi-Box which is connected through USB to a 3D-printer.

![doodle3d-infographic](https://cloud.githubusercontent.com/assets/156066/15449314/4d6f3f14-1f7a-11e6-97fa-275a34164bb2.png)

**Comment by peteruithoven** [comment]
Most docs currently live here: 
http://www.doodle3d.com/help/wiki/firmware
http://www.doodle3d.com/help/wiki/serverside

This issue is about moving such docs to this repo's wiki?

**Comment by companje** [comment]
Yes, eventually I would like to move all documentation from http://www.doodle3d.com/help/wiki to the wikis of the corresponding repository (ie. all print3d related info to the print3d github wiki etc).
